fe/fe-core/src/main/java/com/starrocks/backup/BackupMeta.java

```diff
@@ -63,6 +65,8 @@ public class BackupMeta implements Writable, GsonPostProcessable {
     private Map<String, Table> tblNameMap = Maps.newHashMap();
     // tbl id -> tbl
     private Map<Long, Table> tblIdMap = Maps.newHashMap();
+    @SerializedName(value = "functions")
+    private List<Function> functions = Lists.newArrayList();
```

Comment:
better to use a map which maps name to functions ?

Reply:
It seems that a little redundant to same the name because Function Class has already contains its own FunctionName.

Reply:
We need a lookup by name ? for given s specified function to restore ?